### ISSUE TITLE
Don't show the omitted files in the Test Coverage Toolbar

### ISSUE THREAD

**Issue by paduszyk** [microsoft/vscode#230834]
I like the new Test Coverage Toolbar feature. I use it with Python and Coverage.py run via `pytest` and `pytest-cov` plugin.

Coverage.py enables omitting some files using globs (e.g. configuration modules or development scripts; see [docs](https://coverage.readthedocs.io/en/7.6.1/excluding.html#advanced-exclusion)). I think that *NOT* showing those files in the toolbar could be an interesting enhancement.

Sorry if you find this issue biased to Python. I am not so familiar with other languages and testing tools available for them.

**Comment by eleanorjboyd** [comment]
good idea yes! I can do this specifically on the python side. This will need development time so just putting it for community feedback to see how many people are hitting this issue
